### PR TITLE
Updated package.json and Added Server.js

### PR DESCRIPTION
**Pull request by AndCrit**
Added the express dependency into the package.json file. Added the server.js file and initialized with hello world.

### CODE REVIEW

File: my-app/package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.3",
     "@testing-library/user-event": "^12.6.0",
+    "express": "^4.17.1",
```
Comment (bobhare118) [inline my-app/package.json:9]: Green

Comment (SachJay) [reply my-app/package.json:9]: Red

File: my-app/server.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+var express = require('express');
+var app = express();
+
+app.get('/', function (req, res) {
+  res.send('Hello World!');
+});
+
+app.listen(4200, function () {
+  console.log('Example app listening on port 4200!');
+});
```
Comment (SachJay) [inline my-app/server.js:11]: NEW LINE ANKIT, THIS IS UNACCEPTABLE

**Review by SachJay — CHANGES_REQUESTED** [review]

File: my-app/package.json (hunk 1 above)
Comment (SachJay) [inline my-app/package.json:9]: This should be in its own file there should be 2 different package.json, currently the frontend and backend are combined but apparently thats not scalable.
xd myb

**Review by SachJay — CHANGES_REQUESTED** [review]

**Pull request closed without merging**